lib/services/api/session_manager.ts

```diff
@@ -80,7 +80,7 @@ export class SNSessionManager extends PureService {
 
   private async setSession(session: Session) {
     this.session = session;
-    this.apiService!.setSession(this.session);
+    await this.apiService!.setSession(this.session);
```

Comment:
We call `setSession` from `initializeFromDisk`, and `apiService.setSession` writes to disk. So we don't want to write to disk when we are loading from disk.